examples/tests/cypress/support/commands.js

```diff
@@ -46,9 +46,17 @@ Cypress.Commands.add('sendEvent', (eventType, message) => {
   cy.get('#send-button').click();
 });
 
-Cypress.Commands.add('assertReceivedEvents', (element, message) => {
+Cypress.Commands.add('assertEventInEventList', (message) => {
   // assert number and content of received events
-  cy.get(`#${element}`)
+  cy.get(`#eventList > li`, {timeout: 30000})
+    .should('be.visible')
+    .first()
+    .contains(message)
+});
+
+Cypress.Commands.add('assertEventLogHasMatch', (message) => {
```

Comment:
The channels example is failing, because we have here `#${element}`, but there is no such argument, only `message`. So, we should either pass name of the element from tests or set it to `#event-log`.